src/jvm/test/resources/expected_logs/var_handle/varhandle_boolean_representation.txt

```diff
@@ -13,19 +13,19 @@ The following interleaving leads to the error:
-|   VarHandleBooleanRepresentationTest.staticNumber.compareAndSet(false,false): true at VarHandleBooleanRepresentationTest.operation(VarHandleRepresentationTests.kt:219) |
-|   VarHandleBooleanRepresentationTest.staticNumber.set(false) at VarHandleBooleanRepresentationTest.operation(VarHandleRepresentationTests.kt:220)                       |
-|   array.READ: BooleanArray#1 at VarHandleBooleanRepresentationTest.operation(VarHandleRepresentationTests.kt:222)                                                       |
-|   BooleanArray#1[1].compareAndSet(false,false): true at VarHandleBooleanRepresentationTest.operation(VarHandleRepresentationTests.kt:222)                               |
-|   array.READ: BooleanArray#1 at VarHandleBooleanRepresentationTest.operation(VarHandleRepresentationTests.kt:223)                                                       |
-|   BooleanArray#1[1].set(true) at VarHandleBooleanRepresentationTest.operation(VarHandleRepresentationTests.kt:223)                                                      |
-|   result: void                                                                                                                                                          |
-| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|                                                                                Thread 1                                                                                |
+| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| operation()                                                                                                                                                            |
+|   number.READ: false at VarHandleBooleanRepresentationTest.operation(VarHandleRepresentationTests.kt:216)                                                              |
+|   number.compareAndSet(false,false): true at VarHandleBooleanRepresentationTest.operation(VarHandleRepresentationTests.kt:216)                                         |
+|   number.set(true) at VarHandleBooleanRepresentationTest.operation(VarHandleRepresentationTests.kt:217)                                                                |
+|   VarHandleBooleanRepresentationTest.staticNumber.READ: true at VarHandleBooleanRepresentationTest.operation(VarHandleRepresentationTests.kt:219)                      |
+|   VarHandleBooleanRepresentationTest.staticNumber.compareAndSet(true,false): true at VarHandleBooleanRepresentationTest.operation(VarHandleRepresentationTests.kt:219) |
```

Comment:
Why the traces in these tests have changed?

Reply:
`staticNumber` is now restored and multiple `staticNumber.compareAndSet(...)` calls on it return the same value as the 1st invocation (which wasn't the case before this PR)